integrations/hardhat/index.js

```diff
@@ -1,5 +1,7 @@
 const sapphire = require('@oasisprotocol/sapphire-paratime');
 const { extendEnvironment } = require('hardhat/config');
+require('@nomiclabs/hardhat-ethers');
+require('@nomicfoundation/hardhat-chai-matchers');
```

Comment:
These dev dependencies won't be importable when an dependent package installs the production extension. Please move these to the test file.